services/QuillLMS/client/app/bundles/Shared/components/shared/dataTable.tsx

```diff
@@ -69,6 +69,14 @@ interface DataTableState {
   hoveredColIndex: null|number;
 }
 
+const TableCellContentWrapper = ({ link, children, }) => {
```

Comment:
This approach not only solves an issue, but also feels cleaner to me.  I like this kind of modularity.

Reply:
agreed, I like this approach